python/python/glide/async_commands/cluster_commands.py

```diff
@@ -46,7 +55,7 @@ async def info(
         self,
         sections: Optional[List[InfoSection]] = None,
         route: Optional[Route] = None,
-    ) -> TClusterResponse[str]:
+    ) -> TClusterResponse[bytes]:
```

Comment:
Why `bytes`, but not `Union[str, bytes]` or even better `TGlideString` which is `Union[str, bytes]` (similar to `TClusterResponse`)?